assets/locales/en.json

```diff
@@ -168,7 +168,7 @@
   "沖縄県内感染者発生状況": "Occurrence of infection at Okinawa Prefecture",
   "沖縄県内感染者発生状況(本島)":"/map/en/hontou-en.png",
   "沖縄県内感染者発生状況(離島)":"/map/en/ritou-en.png",
-  "{date} 更新": " Last update: {date}",
+  "{date} 更新": " Last updated: {date}",
```

Comment:
The original "update" seems good as a noun.
I'd like to suggest to use one of options.
- Last update: {date}
- Last updated on {date}
- Updated on {date}